packages/x-charts-premium/README.md

```diff
@@ -0,0 +1,26 @@
+# MUI X Pro
+
+This package is the Pro plan edition of the chart components.
+It's part of [MUI X](https://mui.com/x/), an open-core extension of our Core libraries, with advanced components.
+
+## Installation
+
+Install the package in your project directory with:
+
+```bash
+npm install @mui/x-charts-premium
+```
+
+This component has the following peer dependencies that you need to install as well.
+
+```json
+"peerDependencies": {
+  "@mui/material": "^5.15.14 || ^6.0.0 || ^7.0.0",
+  "react": "^17.0.0 || ^18.0.0 || ^19.0.0",
+  "react-dom": "^17.0.0 || ^18.0.0 || ^19.0.0"
+},
+```
+
+## Documentation
+
+Visit [https://mui.com/x/react-charts/](https://mui.com/x/react-charts/) to view the full documentation.
```

Comment:
The Quickstart will need an additional premium option in its install script (with a `TODO: CHARTS-PREMIUM` of course :) )

![image](https://github.com/user-attachments/assets/59889cfb-ecde-4acc-998f-d3a5c76028b9)

Reply:
I think it is there already, isn't it done by this component? https://github.com/mui/mui-x/pull/18331/files#diff-78ebd4adb67b0ab5d013a8c9cf2473c4680a1267bf31f4af257b18d5bba08200